From 713bc7e7e3d9d7c9f78e0716e6b63ce0e68e46aa Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 16 May 2024 14:58:54 +0200
Subject: [PATCH 0825/1695] clk: stm32mp1_rcc: remove unused defines

Remove the unused defines DT_RCC_CLK_COMPAT and DT_RCC_SECURE_CLK_COMPAT

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9ea93fccfda94cbfcf6ae471a73f6ae7cf06ef3a clk: stm32mp1_rcc: remove unused defines]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I8d8deaab6ef52d1c132503e62a3ff5eb7f9be82e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/380316
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/include/drivers/stm32mp1_rcc.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/core/include/drivers/stm32mp1_rcc.h b/core/include/drivers/stm32mp1_rcc.h
index 7c87a4a87..ca75f1953 100644
--- a/core/include/drivers/stm32mp1_rcc.h
+++ b/core/include/drivers/stm32mp1_rcc.h
@@ -550,9 +550,6 @@
 #define RCC_MP_IWDGFZSETR_IWDG1			BIT(0)
 #define RCC_MP_IWDGFZSETR_IWDG2			BIT(1)
 
-#define DT_RCC_CLK_COMPAT		"st,stm32mp1-rcc"
-#define DT_RCC_SECURE_CLK_COMPAT	"st,stm32mp1-rcc-secure"
-
 #ifndef __ASSEMBLER__
 #include <io.h>
 #include <stdbool.h>
-- 
2.47.2

